cs-http-client/src/main/java/io/cloudslang/content/httpclient/build/conn/SSLConnectionSocketFactoryBuilder.java

```diff
@@ -68,25 +68,26 @@ public SSLConnectionSocketFactory build() {
         boolean trustAllRoots = Boolean.parseBoolean(trustAllRootsStr);
 
         SSLContextBuilder sslContextBuilder = SSLContexts.custom();
+        String changeit = "changeit";
+        String javaKeystore = System.getProperty("java.home") + "/lib/security/cacerts";
```

Comment:
@ligiacentea  "/lib/security/cacerts" maybe it can be extracted into a constant.